templates/download/iot/intel-nuc-desktop.html

```diff
@@ -17,21 +17,17 @@ <h1>Install Ubuntu Desktop on the Intel<sup>&reg;</sup> NUC</h1>
       <div class="u-equal-height p-divider">
         <div class="col-6 p-divider__block">
           <h2>Install Ubuntu Desktop</h2>
-          <p>As an alternative to Ubuntu Core, you can install Ubuntu Desktop 16.04 LTS, where you can use your favourite development tools to create and run snaps.</p>
+          <p>Use your favourite development tools, install snaps and access the vast Ubuntu ecosystem with your NUC.</p>
         </div>
         <div class="col-6 p-divider__block">
           <h3>Minimum requirements</h3>
           <ul class="p-list">
-            <li class="p-list__item is-ticked">An Ubuntu SSO account with an SSH key</li>
             <li class="p-list__item is-ticked">An Intel NUC with BIOS updated to the latest version (<a class="p-link--external" href="http://www.intel.com/content/www/us/en/support/boards-and-kits/000005850.html">update instructions</a>)</li>
-            <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (2GB minimum)</li>
-            <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
-            <li class="p-list__item is-ticked">A Mini HDMI to HDMI cable</li>
+            <li class="p-list__item is-ticked">1 USB 2.0 or 3.0 flash drive (4GB minimum for Dawson Canyon NUCs, 2GB for older generations)</li>
             <li class="p-list__item is-ticked">A USB keyboard and a mouse</li>
-            <li class="p-list__item is-ticked">A monitor with VGA or HDMI interface</li>
+            <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
             <li class="p-list__item is-ticked">A VGA or HDMI cable</li>
```

Comment:
a VGA cable isn't going to help you if you succeed with previous point and have a monitor with an HDMI port

Reply:
Thanks for catching it